blueprint-manager/src/sdk/utils.rs

```diff
@@ -60,13 +60,6 @@ pub fn generate_process_arguments(
         format!("--bind-addr={}", gadget_config.bind_addr),
         format!("--bind-port={}", gadget_config.bind_port),
         format!("--url={}", gadget_config.url),
-        format!(
-            "--logger=Blueprint-{blueprint_id}-Service-{service_id}-{}",
-            opt.instance_id.clone().unwrap_or_else(|| format!(
-                "{}-{}",
-                gadget_config.bind_addr, gadget_config.bind_port
-            ))
-        ),
```

Comment:
I forgot to update this. It's called `--log-id` now, btw.

Reply:
Thanks!